config.json

```diff
@@ -35,15 +35,53 @@
       {
         "slug": "log-line-parser",
         "uuid": "e5476046-5289-11ea-8d77-2e728ce88125",
+
         "concepts": [
           "strings"
         ],
         "prerequisites": [
+          "basics",
+        ],
+        "status": "wip"
+      },
+
+      {
+        "slug": "boutique-inventory",
+        "uuid": "9f3a89f2-196b-4d53-8481-1360b565c797",
+        "concepts": [ "blocks", "enumeration" ],
+        "prerequisites": [ "hash" ],
+        "status": "wip"
+      },
+      {
+        "slug": "boutique-inventory-improvements",
+        "uuid": "cf415960-ceff-4a1c-b65a-c4b5b1a80155",
+        "concepts": ["ostruct" ],
+        "prerequisites": [ "enumeration" ],
+        "status": "wip"
+      },
+
+
+      {
+        "slug": "moviegoer",
+        "uuid": "2aa6c375-8a73-4f5f-ac8f-9db22a86e1f6",
+        "concepts": [
+          "basics"
+        ],
+        "prerequisites": [],
+        "status": "wip"
+      },
+
+      {
+        "slug": "simple-calculator",
+        "uuid": "ef6dca29-a990-4a6e-8b2c-4703fd0f751a",
+        "concepts": [
           "basics"
         ],
+        "prerequisites": [],
         "status": "wip"
       },
 
+<<<<<<< HEAD
```

Comment:
Merge conflict markers should be resolved.

Reply:
Done. Thanks.